README.md

```diff
@@ -236,6 +236,16 @@ Deletes specified schema
 | ------------------------- | ------- | ----------------------------------- |
 | `schemaId`                | number  | ID of sechema                       |
```

Comment:
nit: could you please fix `sechema` (should be `schema`)?  I've been staring at it for a while but was hesitant to submit a one-letter MR